ONSdigital/ssdc-rm-ddl
# Attempt at stopping slient errors

# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

  version: 

# Motivation and Context

When we updated the jackson dependencies back in March, it caused a silent error without anyone realising. This PR is attempting to help mitigate that issue in the future by causing the make dev-build command to fail on errors.

# What has changed



- added `set -e` to the build_init and build_groundzero scripts so an error could be seen if there's dependency changes.
# How to test?



- Run `make dev-build` and everything should work as expected
- Change the pom.xml `jackson-databind` dependency to `2.9.9` and run `make dev-build`. You should see the script fail.
# Links



[Trello](https://trello.com/c/KRQoqrU7/)
